client/packages/invoices/src/OutboundShipment/api/hooks/document/useOutboundUpdateTax.ts

```diff
@@ -27,6 +27,7 @@ export const useUpdateOutboundTax = () => {
     [lines.nodes, mutateAsync]
   );
 
+  // Will need to implement this back when invoice line tax has been decided
```

Comment:
Comment is unclear to me, what do you mean with "implement this back"?

Reply:
Added a tax on Invoice directly since implementation for invoice line tax hasn't been decided and was very confusing in UI